approach_mission.md

```diff
@@ -0,0 +1,164 @@
+# Approaching a Task - determining a website's technical structure. 
```

Comment:
lets change the file name as well to `approach_data_structure_task`

How about the title `How to approach a new task, and its underlying data structure`